.prettierignore

```diff
@@ -0,0 +1 @@
+**/*
```

Comment:
Aren't we completely disabling prettier here?

Reply:
Yes. We don't officially use Prettier at Workleap and it conflicts with our eslint plugin. 😕